decidim-accountability/config/locales/en.yml

```diff
@@ -146,6 +162,10 @@ en:
           row_number: Row
           subject: Successful import of results
           success: Successful import of results. You can review the results in the administration interface.
+      import_projects_mailer:
+        import:
+          subject: Successful import of projects
+          success: Successful imported projects to Accountability. You can review the results in the administration interface.
```

Comment:
In the accountability component we don't have "projects", so I would rephrase this slightly:

```suggestion
          success: Successful imported projects to results in the %{component_name} component. You can review the results in the administration interface.
```

Note that you also need to pass the `component_name` interpolation value for the translation with this change.